.github/workflows/amd-offline-tests.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: "ubuntu-latest"
 
     container:
-      image: rocm/rocm-terminal
+      image: ubuntu:22.04
```

Comment:
6 days ago rocm/rocm-terminal:latest was bumped from 5.5 to 5.5.1
I suspect that they changed something so pip3 started to consume a lot more disk space for package installation.

I've tried to use previous version, but for some reason it did not work too :slightly_frowning_face: , so I tried common ubuntu instead of rocm specific image.